CONTRIBUTING.md

```diff
@@ -43,16 +40,16 @@ git push origin feature/your-feature-name
 
 ## Code Style
 
-We use [black](https://github.com/psf/black) for code formatting and [flake8](https://flake8.pycqa.org/) for linting. To ensure your code meets our style guidelines:
+We use [black](https://github.com/psf/black) for code formatting. To ensure your code meets our style guidelines:
```

Comment:
We may want to look into ruff which is also created by the same folks who created uv

Reply:
Unsure if the overlap is perfect...

Reply:
I originally had planned to address this in this PR but decided instead to list it as a follow up item in #532 

There is definitely improvements to how we lint/test, this cleanup is because we currently don't include flake for linting.